common/main/java/com/couchbase/lite/Defaults.java

```diff
@@ -26,8 +26,6 @@ private LogFile() {}
 
         /** Plaintext is not used, and instead binary encoding is used in log files */
         public static final boolean USE_PLAINTEXT = false;
-        @Deprecated
-        public static final boolean USE_PLAIN_TEXT = USE_PLAINTEXT;
```

Comment:
Where did this go?

Reply:
It is not in EE... Should it be?

Reply:
When did it get introduced here?  Has any release gone out with this deprecated constant?

Reply:
It doesn't cost anything to leave it there.  I put it back.